plugins/janus_videoroom.c

```diff
@@ -1671,7 +1541,47 @@ static void janus_videoroom_message_free(janus_videoroom_message *msg) {
 	g_free(msg);
 }
 
+void janus_videoroom_codecstr(janus_videoroom * videoroom, char* audio_codecs, char* video_codecs, int str_len, const char* split)
+{
+	if(audio_codecs){
```

Comment:
Nitpicking again... :smile: 
Please leave a space before the curly brackets: `if(audio_codecs) {` (same for a few other points where I've seen the same thing, e.g., for video_codecs below).